File: JAVA/I2/src/hr/algebra/models/Vehicle.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package hr.algebra.models;

/**
 *
 * @author pasar
 */
public class Vehicle {
    
    private int idVehicle;
    private String vehicleType;
    private String vehicleMake;
    private int yearOfMake;
    private int kilometersTravelled;
    private boolean isAvailable;
    private String vehicleServiceDetails;

    public Vehicle(int idVehicle, String vehicleType, String vehicleMake, int yearOfMake, int kilometersTravelled, boolean isAvailable,String vehicleServiceDetails) {
        this(vehicleType,vehicleMake,yearOfMake,kilometersTravelled,isAvailable, vehicleServiceDetails);
        this.idVehicle = idVehicle;
    }

    public Vehicle(String vehicleType, String vehicleMake, int yearOfMake, int kilometersTravelled, boolean isAvailable, String vehicleServiceDetails) {
        this.vehicleType = vehicleType;
        this.vehicleMake = vehicleMake;
        this.yearOfMake = yearOfMake;
        this.kilometersTravelled = kilometersTravelled;
        this.isAvailable = isAvailable;
        this.vehicleServiceDetails = vehicleServiceDetails;
    }

    

    public int getIdVehicle() {
        return idVehicle;
    }

    public void setIdVehicle(int idVehicle) {
        this.idVehicle = idVehicle;
    }

    public String getVehicleType() {
        return vehicleType;
    }

    public void setVehicleType(String vehicleType) {
        this.vehicleType = vehicleType;
    }

    public String getVehicleMake() {
        return vehicleMake;
    }

    public void setVehicleMake(String vehicleMake) {
        this.vehicleMake = vehicleMake;
    }

    public int getYearOfMake() {
        return yearOfMake;
    }

    public void setYearOfMake(int yearOfMake) {
        this.yearOfMake = yearOfMake;
    }

    public int getKilometersTravelled() {
        return kilometersTravelled;
    }

    public void setKilometersTravelled(int kilometersTravelled) {
        this.kilometersTravelled = kilometersTravelled;
    }

    public boolean getIsAvailable() {
        return isAvailable;
    }

    public void setIsAvailable(boolean isAvailable) {
        this.isAvailable = isAvailable;
    }

    public String getVehicleServiceDetails() {
        return vehicleServiceDetails;
    }

    public void setVehicleServiceDetails(String vehicleServiceDetails) {
        this.vehicleServiceDetails = vehicleServiceDetails;
    }
    
    
    @Override
    public String toString() {
        return "Vehicle{" + "idVehicle=" + idVehicle + ", vehicleType=" + vehicleType + ", vehicleMake=" + vehicleMake + '}';
    }
    
    
}

File: PPPK/Models/Vehicle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PPPK.Models
{
    public class Vehicle
    {
        public int IDVehicle { get; set; }
        public string VehicleType { get; set; }
        public string Make { get; set; }
        public int YearOfMake { get; set; }
        public int Kilometers { get; set; }
        public bool IsAvailable { get; set; }
        public string VehicleServiceDetails { get; set; }
        public override string ToString() => $"ID: {IDVehicle}, Type: {VehicleType}, Make: {Make}, Service:{VehicleServiceDetails}";

        public string VehicleService() => $"";

        public Vehicle()
        {
        }

        public Vehicle(int iDVehicle, string vehicleType, string make, int yearOfMake, int kilometers, bool isAvailable, string vehicleServiceDetails)
            :this(vehicleType,make,yearOfMake,kilometers,isAvailable,vehicleServiceDetails)
        {
            IDVehicle = iDVehicle;
        }

        public Vehicle(string vehicleType, string make, int yearOfMake, int kilometers, bool isAvailable, string vehicleServiceDetails)
        {
            VehicleType = vehicleType;
            Make = make;
            YearOfMake = yearOfMake;
            Kilometers = kilometers;
            IsAvailable = isAvailable;
            VehicleServiceDetails = vehicleServiceDetails;
        }



        /*public Vehicle(int iDVehicle, string vehicleType, string make, int yearOfMake, int kilometers, bool isAvailable)
            :this(vehicleType, make, yearOfMake, kilometers, isAvailable)
        {
            IDVehicle = iDVehicle;
        }

        public Vehicle(string vehicleType, string make, int yearOfMake, int kilometers, bool isAvailable)
        {
            VehicleType = vehicleType;
            Make = make;
            YearOfMake = yearOfMake;
            Kilometers = kilometers;
            IsAvailable = isAvailable;
        }*/


        /*public Vehicle(string vehicleType, string make, int yearOfMake, int kilometers, bool isAvailable, 
            string vehicleServiceDetails) : this(vehicleType, make, yearOfMake, kilometers, isAvailable)
        {
            VehicleServiceDetails = vehicleServiceDetails;
        }*/
    }
}
